app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -115,15 +116,23 @@ def export
       def generate
         @composer = ReportComposer.from_api_params(params)
         if @composer.valid?
-          response = @composer.render(params: params)
-          send_data response, type: @composer.mime_type, filename: @composer.report_filename
+          response.headers['Content-Type'] = @composer.mime_type
+          # https://discuss.rubyonrails.org/t/actioncontroller-live-chokes-at-buf-push/76490
+          response.headers['Cache-Control'] = 'no-cache'
+          response.headers['Last-Modified'] = '0'
+          response.headers['ETag'] = nil
+          response.stream.write("")
```

Comment:
Shouldn't writing an empty string into an IO be essentially a no-op?

Reply:
https://edgeapi.rubyonrails.org/classes/ActionController/Live.html states:
> Calling write or close on the response stream will cause the response object to be committed. Make sure all headers are set before calling write or close on your stream.